tinygrad/codegen/linearizer.py

```diff
@@ -224,7 +224,8 @@ def get_upcast_dim(self, i) -> List[int]:
     should_upcast = self.opts.supports_float4 and (self.bufs[i].dtype in [dtypes.float32, dtypes.float16] or isinstance(self.bufs[i].dtype, ImageDType))
     return [x for x in self.sts[i].unit_stride_axes() if should_upcast and x >= self.shape_len-self.upcasted and self.sts[i].shape[x] > 1]
 
-  def global_load(self, i:int, idxs:Sequence[VariableOrNum], const=None) -> List[Token]:
+  def global_load(self, i:int, idxs:Sequence[VariableOrNum], const=None, load_cache=None) -> List[Token]:
+    load_type = "val" if const is None else "acc"
```

Comment:
Why this line?

Reply:
it would write acc<num>_<num> for a RawConst

Reply:
I don't see how it's different?